modules/insights-operator-downloading-archive.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-operator.adoc
+
+:_content-type: PROCEDURE
+[id="insights-operator-downloading-archive_{context}"]
+= Downloading your Insights Operator Archive
```

Comment:
Is "Insights Operator Archive" a product name. If it is not, you could update "Archive" to lowercase.

Reply:
Fixed